ksql-rest-app/src/main/java/io/confluent/ksql/rest/server/KsqlRestApplication.java

```diff
@@ -262,7 +262,8 @@ protected void registerWebSocketEndpoints(final ServerContainer container) {
                       JsonMapper.INSTANCE.mapper,
                       statementParser,
                       ksqlEngine,
-                      exec
+                      exec,
+                      versionCheckerAgent.getActiveChecker()
```

Comment:
If you take my advice ad switch this to take `Runnable onRequestEvent` or `Event onRequestEvent` then you can replace this with:

```suggestion
                      versionCheckerAgent::updateLastRequestTime
```